Python/initconfig.c

```diff
@@ -1981,7 +1981,11 @@ config_init_import(PyConfig *config, int compute_path_config)
     if (config->use_frozen_modules < 0) {
         const wchar_t *value = config_get_xoption_value(config, L"frozen_modules");
         if (value == NULL) {
-            config->use_frozen_modules = !config->_is_python_build;
+#ifdef Py_DEBUG
+            config->use_frozen_modules = 0;
+#else
+            config->use_frozen_modules = 1;
```

Comment:
@ericsnowcurrently might know — shouldn’t we solve this by initializing that config flag apropos?

Reply:
I have reworked this and it now initializes with 0 if `Py_DEBUG` and 1 otherwise. Tests are also fixed now.